 website2/_header.scss

```diff
@@ -0,0 +1,127 @@
+.header 
+{
+  
+  background-color: #1F2326;
+  margin: auto;
+  color: #ffffff;
+
+
+  &-links 
+  {
+    
+    margin-left: 150px;
+    margin-right: 150px;
+   font-family: "Open Sans", Arial, Helvetica, sans-serif;
+    font-weight: lighter;
+    font-size: small;
+   justify-content: space-between;
+
+     ul
+    {
+
+      display: flex;
+      list-style: none;
+      padding: 0px;
+      margin-top: 0px;
+      margin-bottom: 0px;
+      
+      
+    }
+   
+  }
+}
+
+
+.header-links ul li
+{
+  margin-top: 20px;
+  margin-left: 20px;
+}
+
+.link-holder a
+{
+  text-decoration: none;
+  text-decoration: none;
+  color: #ffffff;
+  font-family: "Open Sans", Arial, Helvetica, sans-serif;
+  line-height: 25px;
+}
+
+.link-holder a:hover
+{
+  color: #0096d2;
+}
+
+.link-holder 
+{
+  width: 1000px;
+}
+
+.link-right 
+{
+  margin-left: auto;
+}
+
+.logo
+{
+  
+  margin-bottom: 10px;
+  margin-left: 0px !important;
+
+}
+
+.link-right {
+  margin-left: auto; 
+    margin-right: -102px;
+}
+.link-right ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+}
+
+.link-right ul li {
+    float: left;
+}
+
+.link-right ul li a, .dropbtn {
+    display: inline-block;
+    color: white;
+    text-align: center;
+   
+    text-decoration: none;
+}
+
+.link-right ul li a:hover, .dropdown:hover .dropbtn {
+    background-color: gray;
+    color: white;
+}
+
+.link-right ul li.dropdown {
+    display: inline-block;
+}
+
+.dropdown-content {
+    display: none;
+    position: absolute;
+    background-color: #f9f9f9;
+    min-width: 160px;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+}
+
+.dropdown-content a {
+    color: black;
+    padding: 12px 16px;
+    text-decoration: none;
+    display: block;
+    text-align: left;
+}
+
```

Comment:
n sass way u can write this like
.dropbtn {
&:focus,
&:hover {
background-color: #d3d3d3;
}
}